.github/workflows/browser-installation-test.yml

```diff
@@ -32,27 +32,20 @@ jobs:
         run: yarn install
 
       - name: Build the package
-        run: export NODE_OPTIONS=--openssl-legacy-provider yarn build
+        run: |
+          export NODE_OPTIONS=--openssl-legacy-provider
+          yarn build
 
       - name: Pack `packages/browser`
         run: |
           cd packages/browser
           npm pack
           echo "PACKAGE_NAME=$(ls *.tgz)" >> $GITHUB_ENV
 
-      - name: Extract package filename
-        run: echo "PACKAGE_NAME=$(ls *.tgz)" >> $GITHUB_ENV
-
       - name: Test installing the package in a fresh environment
         run: |
-          mkdir test-install
-          cd test-install
-          npm init -y
-          npm cache clean --force
-          npm install ../packages/browser/$PACKAGE_NAME
-
-      - name: Check Installed Package Version
-        run: |
-          cd test-install
-          jq '.version' node_modules/@customerio/cdp-analytics-browser/package.json
-
+          rm -rf node_modules
```

Comment:
maybe leave a comment stating that this is important and in correct location, as removing may lead to ci passing when it should fail